triplet/main.py

```diff
@@ -230,6 +238,31 @@ def train(epoch):
                 100. * batch_idx / len(train_loader), loss.data[0]))
 
 
+def test(epoch):
+    model.eval()
+    scores, distances = [], []
+    for batch_idx, (data_a, data_p, label) in enumerate(test_loader):
+        if args.cuda:
+            data_a, data_p = data_a.cuda(), data_p.cuda()
+        data_a, data_p, labels = Variable(data_a, volatile=True), \
+                                 Variable(data_p, volatile=True), Variable(label)
+        out_a, out_p = model(data_a), model(data_p)
+        dists = torch.sqrt(torch.sum((out_a - out_p) ** 2, 1))  # euclidean distance
```

Comment:
@apaszke here `dist()` won't work since it returns one single value and what I want is the euclidean distance between each of the descriptors in the batch